src/api/docs/doc.test.js

```diff
@@ -18,58 +18,36 @@ afterAll(async () => {
 
 describe('GET /apiDocs/', () => {
   it('Should display "Available versions" text on page.', async () => {
-    await page.goto(`${BASE_URL}/apiDocs/`);
+    await page.goto(`${BASE_URL}/`);
```

Comment:
The template literal is unnecessary here.

Reply:
Sorry, I did not understand what you meant. Could you please explain a bit?

Reply:
That line can be 
```js
await page.goto(BASE_URL);
```
and you can add the slash to the BASE_URL var.

Reply:
Change the line to:
```javascript 
await page.goto(BASE_URL);
`````
There is no need for the extra slash, so you don't have to use tempalte literals

Reply:
Ah. That's right. Thanks!